electrum/gui/qt/wizard/server_connect.py

```diff
@@ -18,18 +22,55 @@ class QEServerConnectWizard(ServerConnectWizard, QEAbstractWizard):
     def __init__(self, config: 'SimpleConfig', app: 'QElectrumApplication', plugins: 'Plugins', daemon: 'Daemon', parent=None):
         ServerConnectWizard.__init__(self, daemon)
         QEAbstractWizard.__init__(self, config, app)
-
-        self.setWindowTitle(_('Network and server configuration'))
+        self.window_title = _('Network and server configuration')
 
         # attach gui classes
         self.navmap_merge({
-            'autoconnect': { 'gui': WCAutoConnect },
-            'proxy_ask': { 'gui': WCProxyAsk },
-            'proxy_config': { 'gui': WCProxyConfig },
-            'server_config': { 'gui': WCServerConfig },
+            'welcome': {'gui': WCWelcome, 'params': {'icon': ''}},
```

Comment:
I get the welcome screen on every start, until I manually change the network settings.

1. `$ rm -rf electrum_data/`
2. `$ ./run_electrum -v --testnet -P -g qt`
3. welcome screen opens, click "Finish" (btw, shouldn't it be "Next"?)
4. create a wallet or just quit wizard there
5.  `$ ./run_electrum -v --testnet -P -g qt`
6. welcome screen opens again

See
https://github.com/spesmilo/electrum/blob/b5d8d31723720de1c415a3cce7f3378b7b77d181/electrum/gui/qt/__init__.py#L487-L488
To avoid this, for example, when leaving the welcome screen with "use default settings" checked, we could set NETWORK_AUTO_CONNECT to True or maybe `config.cv.NETWORK_AUTO_CONNECT.get_default_value()`.
Or have another escape clause from the branch, idk.